src/cpu/x64/brgemm/brgemm_utils.cpp

```diff
@@ -126,9 +126,15 @@ void set_isa_impl(brgemm_t *brg) {
                 is_isa_ok(avx512_core_fp16), avx512_core_fp16, is_isa_ok(avx2),
                 avx2);
     } else if (brg->is_bf16) {
-        brg->isa_impl = utils::map(true, isa_undef, is_isa_ok(avx512_core_amx),
-                avx512_core_amx, is_isa_ok(avx512_core_bf16), avx512_core_bf16,
-                is_isa_ok(avx2_vnni_2), avx2_vnni_2);
+        if (data_type::f32 == brg->dt_a && data_type::bf16 == brg->dt_b) {
```

Comment:
I saw `(dt_a=f32, dt_b=bf16)` combination is considered to be `is_f32` case at line 57, why here it is in `is_bf16`  branch instead?

Reply:
Removed this code block